tests/phpunit/includes/object-cache.php

```diff
@@ -2131,6 +2131,11 @@ public function add_global_groups( $groups ) {
 			$groups = (array) $groups;
 		}
 
+		// Allow force ignoring of global groups.
+		if ( is_array( WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS ) ) {
+			$groups = array_diff( $groups, WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS );
+		}
```

Comment:
Why is this check different from the above check (see my notes above)? Here `array_diff()` should only execute if there's data (teeny tiny performance boost). Maybe instead:

```suggestion
		if ( ! empty( WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS ) ) {
			$groups = array_diff( $groups, WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS );
		}
```

Then if the constant is not an array type and not an empty / falsey, then let PHP natively throw the exception to alert the developer of the type mismatch issue.

Reply:
Sounds good to me :-) The `is_array` was from my initial commit whereas the other checks came in after code feedback :-)